src/components/ReviewConductCode/ReviewConductCode.tsx

```diff
@@ -0,0 +1,111 @@
+			<div className='conduct-code'>
+				<HeaderTitle>Review our conduct code</HeaderTitle>
+
+				<p className='intro'>
+					We are a community driven by dedication, friendship, and the passion to build exceptional things. It’s important for all our members to have read and understood
+					a few key rules.
+				</p>
+
+				<div>
+					<p className='nutshell-title text-h6'>In a nutshell:</p>
+					<ol className='nutshell-list'>
+						<li className='nutshell-item'>
+							<span className='number'>1</span>
+							<span className='step-label'>All our members are committed to maintaining a safe, respectful and welcoming space.</span>
+						</li>
+						<li className='nutshell-item'>
+							<span className='number'>2</span>
+							<span className='step-label'>
+								Developers, designers and other tech workers are welcome to collaborate in projects and events for the community.
+							</span>
+						</li>
+						<li className='nutshell-item'>
+							<span className='number'>3</span>
+							<span className='step-label'>The TorontoJS activities are non-profit and meant to foster the tech community.</span>
+						</li>
+					</ol>
+				</div>
+
+				<ul className='dropdown-list'>
+					<li>
+						<details>
+							<summary>
+								<span>TorontoJS Code of Conduct</span>
+								<ArrowDown />
+								<ArrowUp />
+							</summary>
+							<div className='details-content'>
+								Lorem ipsum dolor sit amet consectetur, adipisicing elit. Porro suscipit harum doloremque, quod tempora architecto atque unde corrupti, ipsa in
+								expedita et provident vel dignissimos modi exercitationem neque reiciendis dicta.
+							</div>
+						</details>
+					</li>
+					<li>
+						<details>
+							<summary>
+								<span>Volunteering Agreement</span>
+								<ArrowDown />
+								<ArrowUp />
+							</summary>
+							<div className='details-content'>
+								Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid incidunt quibusdam, temporibus accusamus eos dolore suscipit? Cum excepturi
+								reiciendis quis nam, provident temporibus laboriosam animi molestias harum optio magni amet.
+							</div>
+						</details>
+					</li>
+					<li>
+						<details>
+							<summary>
+								<span>Image Release Form</span>
+								<ArrowDown />
+								<ArrowUp />
+							</summary>
+							<div className='details-content'>
+								Lorem ipsum dolor sit amet consectetur adipisicing elit. Rem impedit consequatur, fugiat sit asperiores, sunt possimus delectus incidunt, iste atque
+								qui doloremque rerum. Ratione mollitia dicta corporis minus est neque.
+							</div>
+						</details>
+					</li>
+				</ul>
+
+				<label className='checkbox-label'>
+					<input
+						type='checkbox'
+						onChange={() => {
+							setAgreementChecked(!agreementChecked);
+						}}
+					/>
+					I agree to TorontoJS’s conduct code and other forms
+				</label>
+
+				<Button
+					isPrimary
+					isLarge
+					disabled={!agreementChecked}
```

Comment:
```suggestion
					type="submit"
```
We don't need extra state because the html form validation will already take care of that for us.

Reply:
Yeah, I agree to use `type="submit"`, but is it possible to set the `disabled` true/false without the extra state?

Reply:
I think we don't need to disable the button, if any of the required fields is filled or checked, then it won't allow for submission